core/services/synchronization/telem/telem_enhanced_ea_mercury.proto

```diff
@@ -26,4 +26,7 @@ message EnhancedEAMercury {
   int64 round=19;
   int64 epoch=20;
   string asset_symbol=21;
+  string observation_benchmark_price_string = 22;
+  string observation_bid_string = 23;
+  string observation_ask_string = 24;
```

Comment:
Why use string instead of a native raw format that doesn't require parsing/printing?

Reply:
There is no strong preference in this direction, there is a slight favor towards strings because they are human-readable and it simplifies the debugging downstream